Extensions/outbox.js

```diff
@@ -39,7 +39,7 @@ XKit.extensions.outbox = new Object({
 
 		$(m_div).append(`
 			<div id="xkit-outbox-info">
-				This extension no longer functions, but you can view and export your saved data here:
+				This extension no longer functions and has been replaced by a standalone extension, <a href="https://github.com/AprilSylph/Outbox-for-Tumblr#readme" target="_blank">Outbox for Tumblr</a>! You can view and export your saved data here:
```

Comment:
I feel like this could mislead people into thinking the export would be compatible with Outbox for Tumblr; it should include some indication that it's only for personal archiving.

(I would _like_ to build a compatible export format, but realistically, I don't know if I'm going to do that and would rather merge this without giving anyone more hope about that than is warranted.)

Reply:
Agreed! I couldn't figure out any good ideas for the copy—any suggestions?

Reply:
```suggestion
				This feature no longer works and has been replaced by <a href="https://github.com/AprilSylph/Outbox-for-Tumblr#readme" target="_blank">Outbox for Tumblr</a>.<br><br>For personal archiving, you can view and export your historical XKit Outbox data here:
```